### ISSUE TITLE
Cemu 2.0 poor windows scaling at anything other than 100%

### ISSUE THREAD

**Issue by BParks21** [cemu-project/Cemu#25]
When Windows scaling is set to anything other than 100% the gui window elements and anything rendered in the window exhibits blurry scaling. The previous experimental versions and the last stable version do not have this issue.

**Comment by teconmoon** [comment]
I have experienced this on other builds, and end up having to set compatibility settings as follows:
![image](https://user-images.githubusercontent.com/46909426/186526935-366c9258-3cb7-4d80-a19d-89d44bd5a9b9.png)

**Comment by BParks21** [comment]
It isn't required to do this on 1.27.1 or 1.26.2f.

**Comment by Darkhost1999** [comment]
> I have experienced this on other builds, and end up having to set compatibility settings as follows: 

This does not work for me.

**Comment by CorentinBrule** [comment]
in windows 11, 2560x1440 screen, with 125% rescale :
![Capture d’écran 2022-08-24 183514](https://user-images.githubusercontent.com/9805733/187180077-348518f4-61ca-48d7-9486-8095e607bdec.png)

**Comment by Darkhost1999** [comment]
I can definitely say every single window visibly resizes on my screen every tickbox is full sized and their labels 100% legible.
The scaling at 720p 100% is now on every window and every character in parity to 4k 250% scaling.

tldr fixed on my machine and whats better is no compatibility settings required.
@BParks21 Is this fixed for you?

**Comment by BParks21** [comment]
No the interface definitely still looks blurry to me when compared to 1.27.1 experimental. This is at 125% scaling 1440p. Also looks blurry at 4k anything other than 100% still. 
![Screenshot (16)](https://user-images.githubusercontent.com/12180630/201220697-badfe3be-171e-4ee5-8702-baa8557b27d6.png)

**Comment by BParks21** [comment]
I made another discovery related to the scaling. May open a separate issue about it. Anyway I noticed that in Breath of the wild frame times/frame delivery gets worse if Windows is set to anything other than 100% scaling. I get frame time stutters when set to 125%, 150%, 175% etc. The only way to fix it is to set Windows scaling to 100% or override the cemu.exe scaling behavior by setting it to application controlled. Neither of these things should be necessary. The user shouldn't have to worry about this.
